### PR TITLE
Add roadmap to documentation

### PR DESCRIPTION
**Pull request by ColCarroll**
This adds the current roadmap to the documentation and fixes the ppcplot there.  FYI @canyon289 .

### CODE REVIEW

File: arviz/plots/ppcplot.py [hunk 1]
```diff
@@ -40,13 +40,16 @@ def ppcplot(data, ppc_sample, kind='kde', mean=True, figsize=None, textsize=None
                     ax=ax_)
             for pps in ppss:
                 kdeplot(pps,
-                        plot_kwargs={'color': 'C5', 'linewidth': linewidth},
-                        fill_kwargs={'alpha': 0.2},
+                        plot_kwargs={'color': 'C5', 'linewidth': 1},
```
Comment (aloctavodia) [inline arviz/plots/ppcplot.py]: if this is set to 1, it will not scale with figsize.

File: doc/roadmap.rst [hunk 2]
```diff
@@ -0,0 +1,80 @@
+Arviz Roadmap
+=============
+
+This is super preliminary and reflects thoughts on what needs to be done
+to get to a stable release as of August 1, 2018.
```
Comment (aloctavodia) [inline doc/roadmap.rst]: This is tomorrow! What about October 1?

File: doc/roadmap.rst [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+Road to v1.0
+------------
+
+I would guess that this all takes ~6ish months to do. The initial PyPI
```
Comment (aloctavodia) [inline doc/roadmap.rst]: six months from now seems like a lot, I will have more time to contribute starting next week. What about two months ( around the first week of October)?

Comment (ColCarroll) [reply doc/roadmap.rst]: 6 months was including getting releases of pymc3 and pystan out. i'm cautiously optimistic we can have something on pypi in a month, and something good on pypi in 2 months, though!

Comment (aloctavodia) [reply doc/roadmap.rst]: Nice!

**Pull request merged**